### ISSUE TITLE
Indexing of Js.t

### ISSUE THREAD

**Issue by Coobaha** [reasonml-editor/reasonml-idea-plugin#166]
plugin version: 
0.75
### Description

```
let oo = {"age": 1};

oo##<cursor>
```

I'd expect to get "age" in the autocompletion list.

Should be very handy when working with graphql_ppx generated modules and parsed responses (related issue https://github.com/reasonml-editor/reasonml-idea-plugin/issues/164)

```
module Test = [%graphql {|
{
  user {
    name
  }
  allUsers {
    id
  }
}
 |}];

let query = Test.make()
let data = query.parse(someJsonString);
```

Autocompletion will work with data and nested objects too
```
data##<cursor> /* will show user and allUsers */
data##user##<cursor>   /* will show name */
```

**Comment by Coobaha** [comment]
I got some basic autocomplete working but not for a deep fields. Will also try to add support to deep records and js obj methods.

**Comment by Coobaha** [comment]
First js.t improvements #171

**Comment by giraud** [comment]
merged in v0.77
